pkg/cmd/server/start/admission.go

```diff
@@ -7,49 +7,58 @@ import (
 
 	"k8s.io/apimachinery/pkg/util/sets"
 	"k8s.io/apiserver/pkg/admission"
+	kubeapiserver "k8s.io/kubernetes/cmd/kube-apiserver/app"
 
 	// Admission control plug-ins used by OpenShift
-	_ "github.com/openshift/origin/pkg/authorization/admission/restrictusers"
-	_ "github.com/openshift/origin/pkg/build/admission/defaults"
-	_ "github.com/openshift/origin/pkg/build/admission/jenkinsbootstrapper"
-	_ "github.com/openshift/origin/pkg/build/admission/overrides"
-	_ "github.com/openshift/origin/pkg/build/admission/secretinjector"
-	_ "github.com/openshift/origin/pkg/build/admission/strategyrestrictions"
-	_ "github.com/openshift/origin/pkg/image/admission"
-	_ "github.com/openshift/origin/pkg/image/admission/imagepolicy"
-	_ "github.com/openshift/origin/pkg/ingress/admission"
-	_ "github.com/openshift/origin/pkg/project/admission/lifecycle"
-	_ "github.com/openshift/origin/pkg/project/admission/nodeenv"
-	_ "github.com/openshift/origin/pkg/project/admission/requestlimit"
-	_ "github.com/openshift/origin/pkg/quota/admission/clusterresourceoverride"
-	_ "github.com/openshift/origin/pkg/quota/admission/clusterresourcequota"
-	_ "github.com/openshift/origin/pkg/quota/admission/runonceduration"
-	_ "github.com/openshift/origin/pkg/scheduler/admission/podnodeconstraints"
-	_ "github.com/openshift/origin/pkg/security/admission"
-	_ "k8s.io/apiserver/pkg/admission/plugin/namespace/lifecycle"
```

Comment:
Do we call NewAdmissionOptions to register the ns lifecycle plugin? There is also RegisterAllAdmissionPlugins in k8s.io/apiserver.